modules/updating-eus-to-eus-upgrade-cli.adoc

```diff
@@ -37,18 +37,18 @@ master   rendered-master-ecbb9582781c1091e1c9f19d50cf836c       True  	  False
 worker   rendered-worker-00a3f0c68ae94e747193156b491553d5       True  	  False
 ----
 
-. Change to the `eus-4.12` channel by running the following command:
+. Your current version is 4.y, and your intended version to update is 4.y+2. Change to the `eus-<4.y+2>` channel by running the following command:
 +
 [source,terminal]
 ----
-$ oc adm upgrade channel eus-4.12
+$ oc adm upgrade channel eus-<y+2>
 ----
 +
 [NOTE]
 ====
 
-If you receive an error message indicating that `eus-4.12` is not one of the
-available channels, this indicates that Red Hat is still rolling out 4.10 to 4.12 EUS updates.
+If you receive an error message indicating that `eus-<4.y+2>` is not one of the
+available channels, this indicates that Red Hat is still rolling out previous EUS version updates.
```

Comment:
I'm not sure about `previous` here.  For example, at the moment [`eus-4.12` is only 4.12.z][1], while we soak 4.11-to-4.12 updates.  But maybe we can drop `previous` and the remainder of the sentence still sufficiently explains that `eus-4.(y+2)` membership may take time?

[1]: https://github.com/openshift/cincinnati-graph-data/blob/9039267a5bbab91831737b0b7601803dd7c780bd/channels/eus-4.12.yaml